backend/application/syson-sysml-import/src/test/resources/ASTTransformerTest/convertAssignmentTest/model.ast.json

```diff
@@ -0,0 +1,398 @@
+{
```

Comment:
This model doesn't seems to add its associated textual sysml file.
Is it intended?
(same remarks for all other similar cases)

Reply:
artefact of my creation script. removed.